go/prometheusexporter/collector/collector.go

```diff
@@ -789,6 +813,22 @@ func (c *Collector) recordJobMetrics(m *metric, labels map[string]string, metric
 	delete(labels, "type")
 }
 
+func (c *Collector) recordExceptionMetrics(m *metric, labels map[string]string, metrics *CollectorMetrics) {
+	for _, ex := range m.exceptions {
+		exLabels := make(map[string]string)
+		exLabels["exception"] = ex
+		exLabels["app"] = labels["app"]
+		exLabels["env"] = labels["env"]
+		exLabels["action"] = labels["action"]
+		exLabels["type"] = labels["type"]
+		exLabels["instance"] = labels["instance"]
```

Comment:
```suggestion
		exLabels["instance"] = ""
```

In other places we explicitly set the instance label to the empty string.